dotnet/samples/KernelSyntaxExamples/Example12_SequentialPlanner.cs

```diff
@@ -236,6 +271,41 @@ private static IKernel InitializeKernelAndPlanner(out SequentialPlanner planner,
         return kernel;
     }
 
+    private static IKernel InitializeKernelWithMemory()
+    {
+        // IMPORTANT: Register an embedding generation service and a memory store. The Planner will
+        // use these to generate and store embeddings for the function descriptions.
+        var kernel = new KernelBuilder()
+            .WithLogger(ConsoleLogger.Logger)
+            .WithAzureChatCompletionService(
+                TestConfiguration.AzureOpenAI.ChatDeploymentName,
+                TestConfiguration.AzureOpenAI.Endpoint,
+                TestConfiguration.AzureOpenAI.ApiKey)
+            .WithAzureTextEmbeddingGenerationService(
+                TestConfiguration.AzureOpenAIEmbeddings.DeploymentName,
+                TestConfiguration.AzureOpenAIEmbeddings.Endpoint,
+                TestConfiguration.AzureOpenAIEmbeddings.ApiKey)
+            .WithMemoryStorage(new VolatileMemoryStore())
+            .Build();
+
+        return kernel;
+    }
+
+    private static ISemanticTextMemory GetMemory(IKernel? kernel = null)
```

Comment:
You don't have a code paths that uses `GetMemory` with the kernel as a parameter, suggest removing this from the code (boiler plate)

Reply:
I wanted to demonstrate the options for users. They can either retrieve the Memory from their kernel instance or they can initialize their own. Since this is a sample for demonstration purposes, I'm not gonna worry about lean code too much.